src/ddbToEs/index.ts

```diff
@@ -63,5 +63,7 @@ export async function handleDdbToEsEvent(event: any) {
 
         logger.error('Failed to update ES records', e);
         throw e;
+    } finally {
+        await ddbToEsHelper.closeConnection();
```

Comment:
This does fix the issue but I don't think it is the best solution.

What happened with https://github.com/awslabs/fhir-works-on-aws-persistence-ddb/pull/69#discussion_r618701347? Having exactly one instance of the ES client would fix this issue, and give the best performance since you'd still get the benefit of connection reuse.

As it stands now, the lambda function is still initializing an ES client for every invocation, and that's a bad practice.

Reply:
I tried using codes from #69 but it was throwing errors so I switched back to mainline. I can try it out again. But with this fix, the ddbToEs Lambda is no longer the bottleneck.

Reply:
Just finished testing changes in #69, it does fix the issue, but I decided to stick to this fix as it does not bring breaking changes.